graylog_hook_test.go

```diff
@@ -29,6 +29,7 @@ func TestWritingToUDP(t *testing.T) {
 	msgData := "test message\nsecond line"
 
 	log := logrus.New()
+	log.SetReportCaller(true)
```

Comment:
By default `ReportCaller`  if false, we should keep this behavior in that test.
The new behavior is tested bellow already.

Reply:
fixed.